src/styles/buy.css

```diff
@@ -0,0 +1,19 @@
+.buy-link {
+  display: inline-block;
+  background: #00acdc;
+  border-radius: 5px;
```

Comment:
There is no border mentioned in the CSS in default styling, 
there should be transparent border as a default and just change the color in hover state.

Reply:
Got this